tests/acceptance/bootstrap/SettingsContext.php

```diff
@@ -587,7 +588,7 @@ public function getBodyForNotificationSetting(string $user): array {
 			)->getBody()->getContents()
 		));
 		foreach ($settingsValues->bundles[0]->settings as $settingsValue) {
-			if ($settingsValue->name === "disable-email-notifications") {
+			if ($settingsValue->displayName	 === $this->getDisplayNameFromEvents($event)) {
```

Comment:
seems some problem in linter still exist. Please list linter issue in QA repo issue so that we can work later on
```suggestion
			if ($settingsValue->displayName === $this->getDisplayNameFromEvents($event)) {
```